app/src/examples/WithClampExample.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withSpring,
+} from 'react-native-reanimated';
+import { View, Text, Button, StyleSheet, ViewStyle } from 'react-native';
+
+const VIOLET = '#b58df1';
```

Comment:
This example is very similar to https://github.com/software-mansion/react-native-reanimated/pull/5239/files#r1381390672
Once one of this two PRs gets merged (this one and #5195 creating Higher Order Animation WithClamp) I should unify these examples.